bsdate.go

```diff
@@ -194,11 +194,27 @@ func NewFromGregorian(gregorianDay, gregorianMonth, gregorianYear int) (Date, er
 	var bsYear = gregorianYear + 56         //first rough calculation, might become 57 later
 	var bsMonth = 9                         //Jan 1 always fall in BS month Paush which is the 9th month
 	var daysSinceJanFirstToEndOfBsMonth int //days calculated from 1st Jan till the end of the actual BS month,
-	                                        // we use this value to check if the gregorian Date is in the actual BS month
+	// we use this value to check if the gregorian Date is in the actual BS month
```

Comment:
why this change?

Reply:
fixed. It happened when I used the auto indent